zebra-chain/src/primitives/zcash_history.rs

```diff
@@ -0,0 +1,180 @@
+//! Contains code that interfaces with the zcash_history crate from
+//! librustzcash.
+
+// XXX: remove before completing PR
+#![allow(dead_code)]
```

Comment:
Just tagging this TODO, so it shows up in the unresolved comments.

Reply:
It seems if there is only a test calling the function, it still counts as dead code. How should I handle this, since this PR will only have part of the work and the functions won't be called for now?

Reply:
Yep, in the debug and release builds it will be dead code, because they don't build tests.

You can change the TODO to reference the follow-up tickets, and leave it in as a comment.